airbyte-config/models/src/main/java/io/airbyte/config/AirbyteProtocolConverters.java

```diff
@@ -40,15 +41,15 @@
 // todo (cgardens) - hack, remove after we've gotten rid of Schema object.
 public class AirbyteProtocolConverters {
 
-  public static AirbyteCatalog toCatalog(Schema schema) {
```

Comment:
we only ever need to convert from schema into a configured catalog (never the R/O one).